### PR TITLE
Date export script

### PR DESCRIPTION
**Pull request by eddierubeiz**
We do not currently have any plans to change any of the dates in the new digital collections ap to bring the created-date and modified-date sorts closer in line with Sufia.

However, this script outputs out the relevant dates so that, if we change our minds, we have them ready to go. (It's much easier to get at them now, while the old app is still running, than extracting them from the fedora export.)

### CODE REVIEW

File: app/exporters/dump_sufia_dates.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      f.puts "," if comma
+      comma = true
+      tempHash = {
+        date_modified: to_utc(x.date_modified),
+        date_uploaded: to_utc(x.date_uploaded),
+        create_date: to_utc(x.create_date.utc),
+        type: x.class.to_s,
+        id: x.id
+      }
+      f.puts tempHash.to_json
```
Comment (jrochkind) [inline app/exporters/dump_sufia_dates.rb]: If this is meant to be parseable as json, I think you need to put a comma after each tempHash.to_json...

Alternately you could skip the surrounding `{"date_list":['` stuff, and just call it ["newline-delimited json".](http://ndjson.org/) -- if you make sure `tempHash.to_json` is a single line with no newlines!

Or maybe I'm wrong (this is just from a code-read, I didn't try to parse it), and everything is fine?

Let's make sure we've tested we can actually read this stuff before merging.  Could be manual (maybe you've already done it and can just tell me), or we could figure out a way to unit test it somehow.

Comment (eddierubeiz) [reply app/exporters/dump_sufia_dates.rb]: Hey -- the json parses fine:
```
bin/rails  runner dump_sufia_dates.rb
cat  sufia_dates.json   |  python -m json.tool
```
will parse and output a prettified version of it.

Comment (jrochkind) [reply app/exporters/dump_sufia_dates.rb]: OK, can't figure out how that's working, either there are commas I wans't aware of, or the python tool can handle the fact that they are illegally missing, but either way I guess that works, and is documented here as a way to turn it into good json.

**Comment by jrochkind** [comment]
@eddierubeiz says:

> Hey -- the json parses fine:

>      bin/rails  runner dump_sufia_dates.rb
>      cat  sufia_dates.json   |  python -m json.tool
> will parse and output a prettified version of it.

**Pull request merged**